java/google/registry/module/backend/BackendComponent.java

```diff
@@ -69,6 +70,7 @@
         UrlFetchTransportModule.class,
         UseAppIdentityCredentialForGoogleApisModule.class,
         VoidDnsWriterModule.class,
+        FlowInjectionModule.class,
```

Comment:
Hmm. My understanding based on our previous discussions (mostly but not entirely with Justine) was that we would eventually want to copy the service-level components into our own repository and java package, and all customization of the registry would be done there. While that idea isn't fully working at the moment, it seems to me that we'd still on principal like to avoid sprinkling our changes around the source tree, and making changes to `FlowComponent` would do that.

Perhaps I'm not understanding the modularization/customization goal or mechanism well enough?